src/telemetry/telemetry.ts

```diff
@@ -0,0 +1,61 @@
+import { ReactPlugin, withAITracking } from '@microsoft/applicationinsights-react-js';
+import { ApplicationAnalytics, ApplicationInsights } from '@microsoft/applicationinsights-web';
+import { ComponentType } from 'react';
+import ITelemetry from './ITelemetry';
+
+class Telemetry implements ITelemetry {
+  private appInsights: ApplicationInsights;
+  private config: any;
+
+  constructor() {
+    const areWeInDev = process.env.NODE_ENV === 'development';
+
+    this.config = {
+      instrumentationKey: process.env.REACT_APP_INSTRUMENTATION_KEY,
+      disableExceptionTracking: true,
+      disableTelemetry: areWeInDev ? false : true,
+    };
+
+    this.appInsights = new ApplicationInsights({
+      config: this.config
+    });
+  }
+
+  public initialize() {
+    this.appInsights.loadAppInsights();
+
+    this.appInsights.trackPageView();
+  }
+
+  public trackEvent(eventName: string, payload: any) {
+    if (!this.valid(eventName)) {
+      throw new Error('Invalid telemetry event name');
+    }
+
+    this.appInsights.trackEvent({ name: eventName }, payload);
+  }
+
+  public trackException(error: Error) {
+    const exception: any = { error: error.stack };
+    this.appInsights.trackException(exception);
```

Comment:
`trackException` takes `Error` as a param: https://github.com/microsoft/ApplicationInsights-JS#sending-telemetry-to-the-azure-portal and parse it into [ExceptionDetails ](https://github.com/microsoft/ApplicationInsights-JS/blob/18162ec13b07a4839b96e4642ad04b8f9d745ec4/shared/AppInsightsCommon/src/Telemetry/Exception.ts#L120) and put into `data`->`baseData`->`exceptions` bag, so no need to take stack out.